Example/App.tsx

```diff
@@ -41,7 +41,7 @@ const SCREENS: Record<
   string,
   {
     title: string;
-    component: () => JSX.Element;
+    component: () => React.JSX.Element;
```

Comment:
Why do we need `React` prefix here now?

Reply:
That's what the upgrade helper suggested: https://react-native-community.github.io/upgrade-helper/?from=0.72.4&to=0.74.1#RnDiffApp-App.tsx